assets/javascripts/radar.js

```diff
@@ -1,21 +1,49 @@
-spiderChartFramework()
-distributionChartFramework()
+function createVulnerabilityMap(data) {
+  const colors = d3.scaleLinear()
+    .domain([0, 1])
+    .range(['#2c7bb6', '#d7191c']);
+  const parkingMap = d3.select('.climate-map');
```

Comment:
Somehow this got defaulted back to `parkingMap` from `vulnerabilityMap`

Reply:
Argh, must have copied and pasted from the wrong revision when I was moving over. Good catch.